pal/screen-adapter/native/screen-adapter.ts

```diff
@@ -28,9 +28,13 @@ class ScreenAdapter extends EventTarget {
 
     public get windowSize (): Size {
         const dpr = this.devicePixelRatio;
+        // @ts-expect-error interface for OH only
+        const width = globalThis.oh ? jsb.device.getInnerWidth() : window.innerWidth;
+        // @ts-expect-error interface for OH only
+        const height = globalThis.oh ? jsb.device.getInnerHeight() : window.innerHeight;
```

Comment:
Why OH is different?

Reply:
because window.innerWidth window.innerHeight is set using eval function in native.
But eval is not supported on OpenHarmony, so we provide `jsb.device.getInnerXXX` interface

Reply:
> But eval is not supported on OpenHarmony

Will OH support `eval` in future?

Reply:
no， they won't